api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -92,19 +85,16 @@ public ResponseEntity<Runtime> getRuntime(String workspaceNamespace) {
 
     DbWorkspace dbWorkspace = workspaceService.lookupWorkspaceByNamespace(workspaceNamespace);
     String googleProject = dbWorkspace.getGoogleProject();
-    try {
```

Comment:
Used to try to find a runtime with list, but that was only to find deleted runtimes, so now only get is ued.